sqlx-core/src/sqlite/connection/mod.rs

```diff
@@ -33,6 +34,13 @@ pub struct SqliteConnection {
     scratch_row_column_names: Arc<HashMap<UStr, usize>>,
 }
 
+impl SqliteConnection {
+    /// Returns the underlying sqlite3* connection handle
```

Comment:
I feel like we need to list some do-s and don't-s for this pointer but I'm not sure what those should be.

Reply:
The obvious two that come to mind are not to close the connection or free the pointer. I don't think the Rust docs for similar things spell out the do-s and don't-s.